### ISSUE TITLE
Question about pFedMeOptimizer.

### ISSUE THREAD

**Issue by siddharthdivi** [CharlieDinh/pFedMe#2]
[https://github.com/CharlieDinh/pFedMe/blob/96863e05e799fb6ad23248c63824b0381d2bec11/FLAlgorithms/optimizers/fedoptimizer.py#L64](https://github.com/CharlieDinh/pFedMe/blob/96863e05e799fb6ad23248c63824b0381d2bec11/FLAlgorithms/optimizers/fedoptimizer.py#L64)
Can you please explain the update in this line? I'm not sure what 'mu' signifies here, as it is not present in the algorithm given on page 5 of the corresponding paper?

**Comment by CharlieDinh** [comment]
Hi siddharthdivi,
Mu in here is a kind of regularization inside f_i(\theta).
For example: The local optimization problem for each client is defined as $theta_i = min( f_i(\theta_i) + \frac{\lamda}{2}(theta_i -w))$. I can consider the f_i(\theta_i) is cross entropy loss function with regularization $\frac{mu}{2}(theta_i)^2$ inside. So here $\mu$ is regularization term to make f_i function becomes strongly-convex

**Comment by siddharthdivi** [comment]
Thanks for the explanation!
